src/Compilers/Core/Portable/CodeGen/ILBuilderEmit.cs

```diff
@@ -86,7 +86,7 @@ internal void EmitArrayBlockInitializer(ImmutableArray<byte> data, SyntaxNode sy
             var initializeArray = module.GetInitArrayHelper();
 
             // map a field to the block (that makes it addressable via a token)
-            var field = module.GetFieldForData(data, syntaxNode, diagnostics);
+            var field = module.GetFieldForData(data, alignment: 1, syntaxNode, diagnostics);
```

Comment:
Just to confirm, the reason we're using alignment=1 here is because `System_Runtime_CompilerServices_RuntimeHelpers__InitializeArrayArrayRuntimeFieldHandle` doesn't care about alignment, correct?
Similarly, in next method, because of `ILOpCode.Cpblk`?

Reply:
Yes.  I can change it if you'd like, but this will likely cause many tests to need to be updated because doing so will change the generated field name.  If we want to go with https://github.com/dotnet/roslyn/pull/61414#discussion_r890169844 instead, it'd be easier to make this change for consistency even if it's not needed.

Reply:
I'm fine to leave alignment=1 for scenarios that don't need special alignment. Maybe add a comment and add a test that captures that behavior explicitly.